packages/aws-cdk/lib/api/cxapp/exec.ts

```diff
@@ -103,7 +108,9 @@ export async function execProgram(aws: SdkProvider, config: Configuration): Prom
       //   anyway, and if the subprocess is printing to it for debugging purposes the
       //   user gets to see it sooner. Plus, capturing doesn't interact nicely with some
       //   processes like Maven.
-      const proc = childProcess.spawn(commandLine[0], commandLine.slice(1), {
+      const command = commandAndArgs[0];
```

Comment:
What happens if `commandAndArgs[0]` is `mvn package`

Reply:
I think we just need to accept a single string and pass it down to spawn(). Since you use shell:true this should just work without splitting to arguments imho

Reply:
We need to split the arguments because of the other usage of `exec()`, namely this line:

`await exec(commandLine);`

`commandLine` is defined by 

`const commandLine = await guessExecutable(appToArray(app));`

`guessExecutable()` needs a `string[]`, not a `string`. Do you want me to rework this so that we don't need to have any `string[]`s passed to `exec()`, and instead make `exec()` take just a `string`? `exec()` previously operated on a `string[]`.

Reply:
@eladb if `commandAndArgs[0]` is `mvn package` then `spawn()` will still correctly start the process.

Reply:
Seems a bit messy but if this works as is i am okay with that

Reply:
@eladb just made this cleaner, now `mvn package` will be split into `["mvn", "package"]` instead of the previous `["mvn package"]`. Would appreciate another pass on the PR!